kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -464,9 +465,10 @@ export class BrowserInterface {
     }
   }
 
-  public KillPortableExperience(data: { portableExperienceId: string }) {
-    stopPortableExperienceWorker(data.portableExperienceId)
-    unityInterface.UnloadScene(data.portableExperienceId)
+  public async KillPortableExperience(data: { portableExperienceId: string }): Promise<void> {
+    const portableExperienceUrn: PortableExperienceUrn = `urn:decentraland:off-chain:static-portable-experiences:${data.portableExperienceId}`
```

Comment:
- the `portableExperienceId` naming is inconsistent in other parts of the code, is it the same as peId?
- I think the "peId" should be the whole URN 👾